packages/muelu/src/MueCentral/MueLu_Hierarchy_def.hpp

```diff
@@ -183,7 +205,7 @@ namespace MueLu {
         return 0.0;
       }
 
-      totalNnz += as<double>(Am->getGlobalNumEntries());
+      totalNnz += as<double>(getFakeGlobalNumEntries(*Am));
```

Comment:
If I understand the logic of `getFakeGlobalNumEntries`, it is basically assuming proc 0's number of nonzeros is representative of all procs.   My fear is that this could be inaccurate.  If the global number of nonzeros isn't available because the underlying matrix is a Tpetra::CrsMatrix, I would prefer that this calculation just not be done.

Reply:
I think the point of this is just to make `describe()` work.

Reply:
I'm glad this got looked at: There was a change in the default for the calculation of global constants, which broke the describe. These patches to MueLu have been reverted entirely. 

It might be a good idea to have any test that uses the Describe functionality to be expected to call computeGlobalConstants

Reply:
Couldn't `describe` itself call `computeGlobalConstants` if it needs to?